core/governance/market_cp_restore_test.go

```diff
@@ -90,6 +93,41 @@ func TestMarketRestoreFromCheckpoint(t *testing.T) {
 	}
 }
 
+func TestMarketRestoreFromCheckpointWithEmptySuccessor(t *testing.T) {
+	t.Skipf("Skipping test as need to regenerate testcp/checkpoint.cp with appropriate values for LP - Zohar to fix")
```

Comment:
Probably want to unskip it?

Reply:
indeed, I should've known things passing first time around was too good to be true :)